lte/gateway/python/integ_tests/s1aptests/util/traffic_messages.py

```diff
@@ -96,15 +94,13 @@ def __init__(self, is_uplink, is_udp, duration, ip, port):
 
     def __repr__(self):
         ''' String representation of this test instance '''
-        return ' '.join((
-            '%s:' % type(self).__name__,
-            'UPLINK' if self.is_uplink else 'DOWNLINK',
-            'UDP' if self.is_udp else 'TCP',
-            'test,',
-            '%d seconds' % self.duration,
-            'for test device at',
-            '%s:%d' % (self.ip.exploded, self.port),
-        ))
+        link_type = "UPLINK" if self.is_uplink else "DOWNLINK"
+        protocol = "UDP" if self.is_udp else "TCP"
+        return (
+            f'{type(self).__name__}: {link_type} {protocol} test '
```

Comment:
Nit: there is a comma missing here
```suggestion
            f'{type(self).__name__}: {link_type} {protocol} test, '
```

Reply:
nice catch, changed